### DISCUSSION TITLE
Can mlxtend show the features' name in Sequential Feature Selection for Regression

### DISCUSSION THREAD

**Discussion by lewisscola** [rasbt/mlxtend#809] [Q&A] [answered]
Hi,
I see the 'Example 5 - Sequential Feature Selection for Regression' in http://rasbt.github.io/mlxtend/user_guide/feature_selection/SequentialFeatureSelector/#example-5-sequential-feature-selection-for-regression, 
Can I show the most important features' name in mlxtend?(it only show the features' number but not the name)

Thank you!

**Comment by rasbt** [comment, marked as answer]
Yeah, you can use a pandas DataFrame like in Example 12: http://rasbt.github.io/mlxtend/user_guide/feature_selection/SequentialFeatureSelector/#example-12-using-pandas-dataframes

Then, suppose your "fitted" SFS is called `sfs1` and you are interested in the names of the features corresponding to the 3-sized feature set, you can access the names via `sfs1.subsets_[3]['feature_names']`

**Reply by lewisscola** [reply]
Thank you so much!
